data/human/human missions.txt

```diff
@@ -4925,3 +4925,147 @@ mission "Riley"
+			`	The voice of the captain of the Golden Jewel shouts out towards you. "You were unwise to get involved in our business. Hand over that pompous salesman James and I'll let you live."`
+			choice
+				`	"Uhhh... hand over who?"`
+					goto who
+				`	"I don't negotiate with pirates!"`
+					goto negotiate
+				`	"You can take him, but I don't know where he is."`
+					goto who
+			label who
+			`	"Bah, you lying buffoon! I'll find him in your ship's wreckage!"`
+				goto power
+			label negotiate
+			`	"Aye, then I won't negotiate with you!"`
+				goto power
+			label power
+			`	The Vanguard powers up its weapons.`
+	npc kill
+		government "Independent (Killable)"
+		personality heroic vindictive staying
+			confusion 20
+		system "Moktar"
+		ship "Vanguard" "Golden Jewel"		
+		conversation 
+			`The <npc> has been destroyed. You receive a hail from a small escape pod. "Thank you, Captain, you've saved my life. I'm James, meet me on <destination> to receive your reward."`
+			choice
+				`	"Sounds good."`
+					accept
+				`	"What's the deal with you and that pirate?"`
+					goto piratedeal
+			label piratedeal
+			`"That pirate was a potential investor in the firm I work for. We had a disagreement over the terms and it obviously got quite out of hand. I'll need to find a new business partner now..." He trails off. Time to go to <destination>.`
+	on complete
+		"reputation: Independent (Killable)" = 10
+		conversation
+			`When you land on <destination>, James is waiting for you at the spaceport. "Thank you, truly. I'm James Vendeur, Investor Relations Associate with Maralex Controls. Anyway, I've managed to scrounge up my savings. It's not much to someone like you, I'm afraid. Just over ninety thousand credits.`
+			`	However, I see you're clearly a captain of means. I'd have to pull a few strings, but what if instead I get you into a private placement of Maralex stock? To purchase 1 share of Maralex Controls I will need 1 million credits and for you to sign a few documents. Our historical returns are 20% per annum which we expect to continue into perpetuity, paid out to you as a daily dividend of 500 credits."`
+			choice
+				`	"Sure, I'll invest. What do I need to do?"`
+					goto invest
+				`	"Maralex Controls? I thought you were part of the Syndicate."`
+					goto maralex
+				`	"Private placement? Does that mean I own part of the Syndicate?"`
+					goto placement
+				`	"No thanks, I'll take the money please."`
+					goto payup
+			label maralex
+			`	"Maralex Controls is a partially-owned subsidiary of Syndicated Systems that accepts some private outside investment. We design, manufacture, and integrate innovative micro-control solutions for high-performance commercial and military applications for vehicles traveling by air and space. Succinctly, we make the airframes of small planes and spacecraft. Would you like to invest?"`
```

Comment:
```suggestion
			`	"Maralex Controls is a partially-owned subsidiary of Syndicated Systems that accepts some private outside investment. We design, manufacture, and integrate innovative micro-control solutions for high-performance commercial and military air and space vehicles. Succinctly, we make the airframes of small planes and spacecraft. Would you like to invest?"`
```

(too many instances of the word "for" in one sentence)

Reply:
I agree, you've improved the wording.